lib/bap_types/bap_ir_graph.ml

```diff
@@ -143,13 +204,22 @@ module Node = struct
               match succ_tid_of_jmp jmp with
               | None -> None
               | Some tid when tid <> Term.tid dst -> None
-              | Some tid -> Some {src; pos; dst}))
+              | Some _ -> Some {src; pos; dst}))
 
-  let outputs src t : edge seq =
+  let outputs_from_jmps src t : edge seq =
     Term.enum jmp_t src |> Seq.filter_mapi ~f:(fun pos jmp ->
         succ_of_jmp t.sub jmp >>= fun dst ->
         if mem src t && mem dst t then Some {src; pos; dst} else None)
 
+  let outputs src t : edge seq =
```

Comment:
This is the part of the commit that actually bothers me. The successors from the attribute "succs" is used only if no successors are found in the jmp terms. The position is just marked as 0. Although this works in the few cases I am concerned with, this is not a general solution.